include/onnxruntime/core/common/common.h

```diff
@@ -283,6 +283,8 @@ inline std::wstring ToWideString(const std::wstring& s) { return s; }
 inline std::string ToWideString(const std::string& s) { return s; }
 #endif
 
+#define MAX_STR_LEN 2048
```

Comment:
Can we use constexpr instead of a macro? Should try to avoid macros as much as possible.

Reply:
Sure, switched to constexpr.